src/LinuxTracing/include/LinuxTracing/UserSpaceInstrumentationAddresses.h

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) 2021 The Orbit Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef LINUX_TRACING_USER_SPACE_INSTRUMENTATION_ADDRESSES_H_
+#define LINUX_TRACING_USER_SPACE_INSTRUMENTATION_ADDRESSES_H_
+
+namespace orbit_linux_tracing {
+
+// This interface carries information about memory addresses dedicated to user space
+// instrumentation, in particular the entry trampolines, the return trampoline, and the injected
+// library. It allows querying whether an address belongs to a trampoline, and the map name of the
+// injected library.
+class UserSpaceInstrumentationAddresses {
```

Comment:
If you have a suggestion for a better name, please come forward!

Reply:
I would probably call it `UserSpaceInstrumentationAddressRanges`, but find the other suggestions also fine.

Reply:
I feel kind of equally about all of them so I guess I'll just keep it as it is in the interest of choosing the shortest name. Thanks for the suggestions!